frontend/src/components/general-comparison-table/general-comparison-table.tsx

```diff
@@ -102,15 +103,18 @@ const GeneralComparisonTable: React.FC = () => {
           <div className={styles.tableCell} data-optiontitle="wheeldrive">
             Wheel Drive
           </div>
-          {[...options].map((option) => (
-            <div
-              className={styles.tableCell}
-              data-optiontitle={option}
-              key={uuid4()}
-            >
-              {option}
-            </div>
-          ))}
+          {[...options].map((option) => {
+            if (findEmptyOptions(generalInfo)?.includes(option)) return;
```

Comment:
Move `findEmptyOptions` outside the loop. Because now it redundantly reruns on each loop iteration